airbyte-integrations/connectors/source-okta/source_okta/schemas/logs.json

```diff
@@ -38,11 +35,10 @@
           "type": ["string", "null"]
         },
         "authenticationStep": {
-          "type": ["integer", "null"]
```

Comment:
Why are we removing these nulls? have we confirmed that these fields are really always set?

Reply:
Documentation sets that these fields are not nullable.